### PR TITLE
Commit updates on text CSS and alignment.

### PR DESCRIPTION
**Pull request by ucpasas**
Update text CSS and alignment

### CODE REVIEW

File: buildscripts/pages/gallery.html [hunk 1]
```diff
@@ -29,7 +43,7 @@ <h5 class="d-block d-sm-none">udStream Showcase</h5>
                 <h2 class="d-none d-sm-block d-lg-none">udStream Showcase</h2>
                 <h1 class="d-none d-lg-block">udStream Showcase</h1>
             </div>
-            <div class="col-md-3 d-none d-md-block" style="text-align: right;">
+            <div class="col-md-2 d-none d-md-block" style="text-align: right;">
```
Comment (OsiJr) [inline buildscripts/pages/gallery.html:46]: Doesn't add to 12

File: buildscripts/pages/gallery.html [hunk 2]
```diff
@@ -75,7 +89,8 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
 				name: "Japan",
 				image: "https://models.euclideon.com/demos/thumbs/japan.png",
 				project: "udStream.html?f=euclideon:project/f4cddc9c-6922-4762-8682-94a536dac451",
-				description: "This has some high quality models from Japan."
+				description: "This has some high quality models from Japan.",
+				link: "https://www.euclideon.com/mount-fuji/"
```
Comment (OsiJr) [inline buildscripts/pages/gallery.html:93]: `link` isn't used anywhere in the system

File: buildscripts/pages/gallery.html [hunk 3]
```diff
@@ -95,8 +110,10 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
 				project: "udStream.html?f=euclideon:project/a8a0010b-e72b-4557-a449-fa97b6847639",
 				description: "The Great Barrier is the largest coral reef ecosystem on Earth."
 			}
-		]
+		],
+		selectedCard : -1
```
Comment (OsiJr) [inline buildscripts/pages/gallery.html:114]: This isn't used

File: buildscripts/pages/gallery.html [hunk 4]
```diff
@@ -95,8 +110,10 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
...
 	  }
+
```
Comment (OsiJr) [inline buildscripts/pages/gallery.html:116]: Why extra line here?

**Review by OsiJr — CHANGES_REQUESTED** [review]
Lots of unused things were added?

File: buildscripts/pages/gallery.html (hunk 1 above)
Comment (ucpasas) [reply buildscripts/pages/gallery.html]: Reverted back to 3

File: buildscripts/pages/gallery.html (hunk 2 above)
Comment (ucpasas) [reply buildscripts/pages/gallery.html]: Removed link.

File: buildscripts/pages/gallery.html (hunk 3 above)
Comment (ucpasas) [reply buildscripts/pages/gallery.html]: Removed unnecessary lines

File: buildscripts/pages/gallery.html (hunk 4 above)
Comment (ucpasas) [reply buildscripts/pages/gallery.html]: Removed extra line.

**Review by OsiJr — APPROVED** [review]

**Pull request merged**